src/test/resources/xml/xml-with-namespace-and-empty-tag.xml

```diff
@@ -0,0 +1,5 @@
+<proc:procedure xmlns:proc="urn:com.acrolinx.itms.procedure/1.0">
```

Comment:
even if it is optional, it would be nice to start each XML file with an XML declaration like:

`<?xml version="1.0" encoding="UTF-8"?>`

(and also we usually have a line-break at the end of each file)